README.md

```diff
@@ -42,6 +42,8 @@ See the [documentation](https://soundata.readthedocs.io/) for more examples and
 ### Currently supported datasets
 
 * ESC-50
+* TAU Urban Acoustic Scenes 2019
```

Comment:
I think my message in the PR got merged before you saw it. I'm ok with having this table for now but my experience with `mirdata` is that is better to remove it because 1) we have the table in the docs already and we have to update yet another thing (so unnecessary work); and 2) this list get's pretty ugly as the library grows

Reply:
I agree - I mostly want to have it there for the launch so that people curious about it immediately see that there are many useful datasets already incorporated. Once we've "launched", we can remove this table (or auto-import it from the docs rst file or some other nicer solution)